dietpi/dietpi-software

```diff
@@ -7012,6 +7021,29 @@ _EOF_
 
 		fi
 
+		#Docker
+		INSTALLING_INDEX=162
+		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
+
+			Banner_Installing
+
+			INSTALL_URL_ADDRESS='https://get.docker.com'
+
+			/DietPi/dietpi/func/check_connection "$INSTALL_URL_ADDRESS"
+
+			if (( $? == 0 )); then
+				# Offical Docker recommended install command
+				wget -O DockerInstall.sh https://get.docker.com
```

Comment:
Minor:
You can use the variable
```wget -O DockerInstall.sh "$INSTALL_URL_ADDRESS"```

Reply:
Agreed and amused since I used the variable on the other installs. Will make changes tonight.